File-Formats/DocIO/Working-with-Charts.md

```diff
@@ -2365,16 +2365,122 @@ wordDocument.Close()
 {% endhighlight %}
 
 {% highlight c# tabtitle="UWP" %}
-//DocIO supports chart to image conversion in Windows Forms, WPF, ASP.NET and ASP.NET MVC platform alone.
+//Open the file as Stream.
+using (Stream docStream = typeof(App).GetTypeInfo().Assembly.GetManifestResourceStream("Sample.Assets.Template.docx"))
```

Comment:
//DocIO supports chart to image conversion in Windows Forms, WPF, ASP.NET and ASP.NET MVC platform alone. You can convert chart as image in UWP using DocIORenderer.